bcdaworker/main.go

```diff
@@ -106,10 +107,19 @@ func processJob(j *que.Job) error {
 					return err
 				}
 			}
-			err := os.Rename(oldpath, newpath)
-			if err != nil {
-				log.Error(err)
-				return err
+			// Skipping encryption is NOT the default.  This code will be removed before ATO
```

Comment:
no.  I am asserting that we will definitely NOT ever deliver unencrypted files under any circumstances after we enter that phase.

Reply:
This comment doesn't read that way, since you say 'this code will be removed'. Fix the comment to accurately reflect the intent. Also, if we intend for an environment variable never to be used in prod, we should probably add a condition to the if clause that checks the environment.